locations/spiders/superdry.py

```
@@ -13,6 +13,9 @@ class SuperdrySpider(SitemapSpider, StructuredDataSpider):
     item_attributes = {"brand": "Superdry", "brand_wikidata": "Q1684445"}
     sitemap_urls = ["https://stores.superdry.com/sitemap.xml"]
     wanted_types = ["ClothingStore"]
+    search_for_email = False
+    search_for_facebook = False
+    search_for_twitter = False
     drop_attributes = {"image"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)